CI/physmon/workflows/physmon_track_finding_ttbar.py

```diff
@@ -38,32 +38,38 @@
         Path(__file__).parent.parent / "fpe_masks.yml"
     ) + [
         acts.examples.Sequencer.FpeMask(
-            "Examples/Algorithms/Fatras/src/FatrasSimulation.cpp:172",
+            "Examples/Algorithms/Fatras/src/FatrasSimulation.cpp",
+            (172, 173),
             acts.FpeType.FLTINV,
             1,
```

Comment:
should we also require an issue number on these kind of masks?

Reply:
We could, but it's not trivial to ensure they're valid. For the code comments, there is a specific syntax that I can look for, whereas this here is only accessible from the job run itself. Any ideas?

Reply:
sure!